package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever",
-  "version": "0.11.1",
+  "version": "0.12.1",
```

Comment:
nit - `0.12.0`

I wonder if we should just make this `1.0.0` at this stage

Reply:
I think since it's a non-breaking change that we want folks to pick up automatically, minor version is better.